src/exceptions.py

```diff
@@ -4,17 +4,21 @@
 """Exceptions used by the Flask charm."""
 
 
-class WebserverConfigInvalidError(Exception):
-    """Exception raised when a webserver configuration is found to be invalid.
+class CharmConfigInvalidError(Exception):
```

Comment:
I'd define this exception in `charm_state.py` for higher cohesion

Reply:
The exceptions are used in `charm_state.py` and `webserver.py`